README.md

```diff
@@ -34,11 +34,12 @@ bundle update braintrust
 ```
 
 ## Usage
+The library needs to be configured with your account's secret key which is available in your [Braintrust Dashboard](https://braintrust.dev/app/settings?subroute=api-keys).
```

Comment:
As an alternative to specifying an api key in the code, you may also set the `BRAINTRUST_API_KEY` env variable.

So maybe we could add a statement like
```
You may specify the API key explicitly as a parameter to `Client.new`, or alternatively, set it in the environment variable `BRAINTRUST_API_KEY`.
```